qa/tasks/s3tests.py

```diff
@@ -46,11 +46,12 @@ def download(ctx, config):
         else:
             log.info("Using branch '%s' for s3tests", branch)
         sha1 = cconf.get('sha1')
+        git_remote = cconf.get('git_remote', None) or teuth_config.ceph_git_base_url
         ctx.cluster.only(client).run(
             args=[
                 'git', 'clone',
                 '-b', branch,
-                teuth_config.ceph_git_base_url + 's3-tests.git',
+                git_remote + 's3-tests.git',
```

Comment:
@pritha-srivastava no I don't have a PR, this was just a commit in an older PR, its good to get it in this PR itself (and by *this*, earlier I meant your current PR )